src/web/TextInput.tsx

```diff
@@ -101,11 +101,14 @@ export class TextInput extends React.Component<Types.TextInputProps, TextInputSt
                 />
             );
         } else {
-            return (
+            let keyboardType = this._getKeyboardType().keyboardType;
```

Comment:
It's inefficient and unnecessary to call this method twice. You can use object binding to avoid this:
```
    let { keyboardType, wrapInForm } = this._getKeyboardType();
```